deploy.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+import argparse
+import os
+import shutil
+import signal
+import subprocess
+import sys
+import time
+import zipfile
+
+DEPLOY_DIR = "/home/server-snapshots"
+CURRENT_DEPLOY_DIR = os.path.join(DEPLOY_DIR, "whereuat-server")
+BACKUP_DIR = os.path.join(DEPLOY_DIR, "_whereuat-server")
+TMP_DIR = os.path.join(DEPLOY_DIR, "tmp")
+
+
+# Make a new directory in the deploying directory and return the path of the new
+# directory.
+def makeNewDir():
+  new_dir = "{:0.0f}".format(time.time())
+  new_path = os.path.join(DEPLOY_DIR, new_dir)
+  os.mkdir(new_path)
+  return new_path
+
+
+# Delete the old backup.
+def removeBackup():
+  try:
+    shutil.rmtree(BACKUP_DIR)
+  except OSError as e:
+    if e.errno == 2:
+      print "Backup directory {} doesn't exist. Creating it now.".format(
+        BACKUP_DIR)
+      os.mkdir(BACKUP_DIR)
```

Comment:
I actually don't think we want to make the directory if it doesn't exist because the whole point of this function is to delete the directory anyway.